### DISCUSSION TITLE
/posts/2016-10-21-derivation-of-backpropagation-plus-a-network-diagram

### DISCUSSION THREAD

**Discussion by owenjonesuob** [owenjonesuob/owenjonesuob.github.io#4] [Announcements]
Derivation of backpropagation, plus a network diagram 
 http://owenjonesuob.github.io/maths/derivation-of-backpropagation-plus-a-network-diagram/

**Comment by owenjonesuob** [comment]
```header
Originally posted by meduz on 2020-04-20 11:35:30
````
Hi, thanks a lot for your post!\
is it possible that

`\frac{\partial J}{\partial a} = -y / a + (1-y) / (1-a)`

but in the and \\delta is the same.\
cheers,\
Laurent

**Reply by owenjonesuob** [reply]
```header
Originally posted on 2020-04-20 15:51:31
````
You\'re absolutely right Laurent! It\'s been a long time, but perhaps I
should still fix them\...

I wrote about this more recently, and much more rigorously, in a paper
here which you may be interested in too:
[https://github.com/owenjone\...](https://github.com/owenjonesuob/neural-nets-for-classification/blob/master/report/report.pdf &quot;https://github.com/owenjonesuob/neural-nets-for-classification/blob/master/report/report.pdf&quot;)

**Reply by owenjonesuob** [reply]
```header
Originally posted by meduz on 2020-04-30 07:27:36
````
thanks a lot, very nice and clear report! I did similar experiments with
logistic regression and gradient descent at
[https://laurentperrinet.git\...](https://laurentperrinet.github.io/sciblog/posts/2020-04-08-fitting-a-psychometric-curve-using-pytorch.html &quot;https://laurentperrinet.github.io/sciblog/posts/2020-04-08-fitting-a-psychometric-curve-using-pytorch.html&quot;)
